valhalla/baldr/graph_utils.h

```diff
@@ -6,14 +6,14 @@
 
 #include <boost/property_tree/ptree.hpp>
 
-#include "baldr/graphid.h"
-#include "baldr/graphreader.h"
-#include "baldr/graphtile.h"
-#include "filesystem.h"
-#include "midgard/logging.h"
+#include <valhalla/baldr/graphid.h>
```

Comment:
moved this header from mjolnir to baldr and called it graph_utils.h. it was only used in valhalla_add_elevation.cc (& its test), but could be useful in other places and really shouldn't live inside mjolnir